tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,658 @@
+			}()
+
+			framework.Logf("Scale up sts replica count to 5")
+			scaleUpReplicaCount = 5
+			err = scaleUpStatefulSetPod(ctx, client, statefulset, namespace, scaleUpReplicaCount,
+				true, false)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			framework.Logf("Scale down sts replica count to 1")
+			scaleDownReplicaCount = 1
+			err = scaleDownStatefulSetPod(ctx, client, statefulset, namespace, scaleDownReplicaCount,
+				true, false)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		}
+
+	})
+
+	/* TESTCASE-2
+
+	verify volume lifecycle ops post password rotation
+
+	Steps:
+		1. Create sts on the cluster
+		2. perform password rotation
+		3. scale up and down the sts created in step 1
+		4. Delete all the sts and pvcs which belong to them
+		5. create a PVC and pod with it
+		6. delete the pod and pvc created in step 6
+	*/
+
+	ginkgo.It("[csi-multi-svc] Verify volume lifecycle ops post password rotation", ginkgo.Label(p0, multiSvc, stable), func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var pvclaim *v1.PersistentVolumeClaim
+		var err error
+
+		for i := 0; i <= numberOfSvc-1; i++ {
+			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyNames[i])
+			scParametersList[i][scParamStoragePolicyID] = profileID
+			scParametersList[i][scParamFsType] = ext4FSType
+
+			client := clients[i]
+			namespace := namespaces[i]
+			if i == 1 {
+				os.Setenv(kubeconfigEnvVar, kubeconfig1)
+				framework.TestContext.KubeConfig = kubeconfig1
+				client, err = createKubernetesClientFromConfig(kubeconfig1)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred(),
+					fmt.Sprintf("Error creating k8s client with %v: %v", kubeconfig1, err))
+			}
+			// create resource quota
+			createResourceQuota(client, namespace, rqLimit, defaultNginxStorageClassName)
+
+			scSpec := getVSphereStorageClassSpec(defaultNginxStorageClassName, scParametersList[i], nil, "", "", false)
+			sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+
+			ginkgo.By("Create StatefulSet and verify pv affinity and pod affinity details")
+			service, statefulset, err := createStafeulSetAndVerifyPVAndPodNodeAffinty(ctx, client, namespace,
+				true, 3, false, allowedTopologies, 0, false, false, false, "", "", nil, false)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				fss.DeleteAllStatefulSets(client, namespace)
+				deleteService(namespace, client, service)
+			}()
+
+			// getting supervisorID and password
+			vsphereCfg := getSvcConfigSecretData(client, ctx, csiNamespace)
+			ginkgo.By("Perform password rotation on the supervisor")
+			err = performPasswordRotationOnSupervisor(vsphereCfg.Global.SupervisorID, vcAddress)
```

Comment:
create a util method for it and make it generic

Reply:
created another method